CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ CHANGELOG
 
 ## Not released
 
+## 0.8.3
```

Comment:
Please put it unde the `Not released` section, it will be moved as part of the release process.

Reply:
I am confused. It is below ## Not released

Reply:
remove the `## 0.8.3` line